openlibrary/macros/AffiliateLinks.html

```diff
@@ -6,10 +6,11 @@
 
 $ bwb_affiliate_url = 'https://www.anrdoezrs.net/links/%s/type/dlg/http://www.betterworldbooks.com/' % affiliate_id('betterworldbooks')
 $ amazon_affiliate_url = 'https://www.amazon.com/dp/%s/?tag=%s'
+$ bwb_metadata = {}
+$ amz_price = {}
```

Comment:
Let's default this to none; it looks like it's the literal price, not a dict

```suggestion
$ amz_price = None
```